Voice Note/Views/RecordingListView.swift

```diff
@@ -13,11 +13,11 @@ struct RecordingListView: View {
     var body: some View {
         NavigationStack {
             List {
-                ForEach(voiceNoteViewModel.recordingList) { recording in
+                ForEach((1...10), id: \.self) {
                     ListItem (
                             voiceNote: VoiceNote(
-                                    noteId: recording.id,
-                                    noteTitle: "Note - \(recording.id)",
+                                    noteId: UUID(),
```

Comment:
this will create  a new id. We can not delete or edit note later from DB or document folder.

Reply:
Or its just temp show case of List?